src/components/LogDetailsModal.vue

```diff
@@ -1,3 +1,7 @@
+<!--
+	SPDX-FileCopyrightText: 2023 Ferdinand Thiessen <rpm@fthiessen.de>
```

Comment:
Shouldn't it be changed with a next contributor? Don't want to belittle anyone's efforts, but IMO would be better to unify all copyrights pre-emptive

Reply:
@susnux what do you think? Fine with you? I can do that then.

Reply:
Yes fine with me, I think we should use the Authors.md for keeping track of authors so :heavy_check_mark:

Reply:
Might have seen also a couple of copyrights of Côme and Robin :eyes: 

but if CI will be greeen, I can follow-up this later

Reply:
I think this is fine, we can ask them or simply add the generic one additionally to it.